updates_and_signals/safe_message_handlers/src/client.ts

```diff
@@ -0,0 +1,14 @@
+import { Connection, Client, WorkflowHandle } from '@temporalio/client';
+import { v4 as uuid } from 'uuid';
+
+import { clusterManagerWorkflow } from './workflows';
+
+export async function startClusterManager(): Promise<WorkflowHandle<typeof clusterManagerWorkflow>> {
+  const connection = await Connection.connect({ address: 'localhost:7233' });
+  const client = new Client({ connection });
+  return client.workflow.start(clusterManagerWorkflow, {
+    args: [{}],
```

Comment:
nit: Use a default arg value on the `clusterManagerWorkflow` function instead, and no arg here. The fact that the workflow propagates its state through this arg on CAN is an internal detail that client-side has no need to know in this case.